src/helpers/analyticsEvents/highlighting/CreateNote.ts

```diff
@@ -0,0 +1,23 @@
+import { createSelector } from 'reselect';
+import * as selectNavigation from '../../../app/navigation/selectors';
+import { AnalyticsEvent } from '../event';
+
+const createNote = 'REX highlighting - create note with chosen color';
```

Comment:
its helpful for these to be short, so maybe just 
```suggestion
const createNote = 'REX highlighting - create note';
```